contracts/eosio.system/src/eosio.system.cpp

```diff
@@ -352,6 +367,18 @@ namespace eosiosystem {
       refunds_table.erase( it );
    }
 
```

Comment:
I think the input parameter should be the annual rate since it's more intuitive. The continuous rate is then calculated and saved inside the action.